util/dvsim/utils.py

```diff
@@ -518,3 +518,26 @@ def print_msg_list(msg_list_title, msg_list, max_msg_count=-1):
                 break
         md_results += "```\n"
     return md_results
+
+
+def rm_path(path, ignore_error=False):
+    '''Removes the specified path if it exists.
+
+    'path' is a Path-like object. If it does not exist, the function simply
+    returns. If 'ignore_error' is set, then exception caught by the remove
+    operation is raised, else it is ignored.
+    '''
+
+    try:
+        if os.path.islink(path):
+            os.remove(path)
+        elif os.path.isdir(path):
+            shutil.rmtree(path)
+        else:
+            os.remove(path)
```

Comment:
Doesn't `shutil.rmtree` work on regular files and symlinks too?

Reply:
Passing file to it raises `NotADirectoryError` and passing symlink raises `OSError: Cannot call rmtree on a symbolic link`.

Reply:
Oh! I didn't know that! Thanks.